### ISSUE TITLE
[Antura space] background elements disappear if you tap on them 

### ISSUE THREAD

**Issue by zixofranic** [vgwb/Antura_arabic#679]

**Comment by FrancyCavallari** [comment]
if you are fast enough with your tap to make it happen....
... you get the following error in the Log:
```
NullReferenceException: Object reference not set to an instance of an object
Antura.AnturaSpace.ShopPhotoManager.Start () (at Assets/_app/_scripts/AnturaSpace/Shop/ShopPhotoManager.cs:29)

```

**Comment by CatmanIta** [comment]
Actually, the error is with the debug panel (fixed), investigating the missing tree now...

**Comment by CatmanIta** [comment]
Fixed in [b03fc88]
